### PR TITLE
add social links with icon componet to header

### PR DESCRIPTION
**Pull request by aholtzman**
## Related Issue

https://github.com/ProjectEvergreen/greenwood/issues/233

as well as #232 

## Summary of Changes
created social link component for the header
1. [x] added svgs icons as separate components
2. [x] created a component of the icons with links to respective pages
3 [x] modified header to better adapt to icon component\
4 [x] optimized header for mobile layout

### CODE REVIEW

**Comment by aholtzman** [comment]
Still coming up wonky - got a new Slack icon to work but a new GitHub logo got all cronenberged somehow and the twitter icon steadfastly refuses to show itself on Chrome.

**Comment by aholtzman** [comment]
@thescientist13 I will have this finished up tomorrow - though having valid SVG files not rendering for unknown reasons is something we may want to dig further into (also the 200 character line limit is also a pain with SVG's as they tend to run real long...)

File: www/components/icons/twitter-icon.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { svg } from 'lit-element';
+
+const twitterIcon = svg`<?xml version="1.0" ?>
+<!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'>
```
Comment (thescientist13) [inline www/components/icons/twitter-icon.js]: could we indent this one too?

File: www/components/social-icons/social-icons.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import twitterIcon from '../icons/twitter-icon';
+import slackIcon from '../icons/slack-icon';
+
+class SocialIcons extends LitElement {
+  render() {
+    return html`
+    <style>
+      ${css}
+    </style>
+        <a class="icons" href="https://github.com/ProjectEvergreen" target="_blank" rel=”noreferrer noopener” aria-label="open github page">${githubIcon}</a>
```
Comment (thescientist13) [inline www/components/social-icons/social-icons.js]: should we consider wrapping this in a `<ul>`?

File: www/components/social-icons/social-icons.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+         class="icons slack-icon"
+         target="_blank"
+         rel=”noreferrer noopener”
+         aria-label="slack">${slackIcon}</a>
+        <a href="https://twitter.com/PrjEvergreen" target="_blank" rel=”noreferrer noopener” aria-label="open twitter page">${twitterIcon}</a>
+    `;
+  }
+}
+
+customElements.define('eve-socialicons', SocialIcons);
```
Comment (thescientist13) [inline www/components/social-icons/social-icons.js]: Maybe call it something like **social-share-bar**, and we can delimit the the tag name as `<eve-social-share-bar>`?

**Pull request merged**